ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -208,11 +214,76 @@ StackLayout {
     ColumnLayout {
         spacing: 0
 
-        TopBar {
+        StatusChatToolBar {
             id: topBar
-            z: 60
-            Layout.alignment: Qt.AlignHCenter | Qt.AlignTop
             Layout.fillWidth: true
+
+            property string chatId: chatsModel.channelView.activeChannel.id
+            property string profileImage: appMain.getProfileImage(chatId) || ""
+
+            chatInfoButton.title: {
+              if (chatsModel.channelView.activeChannel.chatType === Constants.chatTypeOneToOne) {
+                  return Utils.removeStatusEns(chatsModel.userNameOrAlias(chatsModel.channelView.activeChannel.id))
+              }
+              return chatsModel.channelView.activeChannel.name
+            }
+            chatInfoButton.subTitle: {
+                switch (chatsModel.channelView.activeChannel.chatType) {
+                    case Constants.chatTypeOneToOne:
+                        return (profileModel.contacts.isAdded(topBar.chatId) ?
+                            profileModel.contacts.contactRequestReceived(topBar.chatId) ?
+                                qsTr("Contact") :
+                                qsTr("Contact request pending") :
+                            qsTr("Not a contact"))
+                        break;
+                    case Constants.chatTypePublic:
+                        return qsTr("Public chat")
```

Comment:
I think we should either have `;` at the ends everywhere in QML or nowhere. The latter is the recommended Qt QML style and my personal preference as well.

Reply:
No strong opinion here. Since JS has ASI, I don't see it as a big deal, but happy to change if there's consensus to add them manually (just means we'll have to touch a lot of files)

Reply:
We don't need to change all code at once. Just if you started modifying a block, then inside that block it is better to make all statements to look identically. Over the time all the code will be eventually updated.

Reply:
Sure thing

>better to make all statements to look identically

Exactly the reason why I've been sticking to no semicolons so far